### PR TITLE
ARTART Review

### PR DESCRIPTION
**Pull request by seanturner**

### CODE REVIEW

**Review by jariarkko — COMMENTED** [review]
All the changes look good, except perhaps the change where to recommendation to phase out legacy AKA without FS was changed to not say "long term".

File: draft-ietf-emu-aka-pfs-latest.xml [hunk 1]
```diff
@@ -1726,7 +1725,7 @@ EMSK     = MK_ECDHE[768..1279]
 
    <t>Best practice security today is to mandate forward secrecy (as
    is done in WPA3, EAP-TLS 1.3, EAP-TTLS 1.3, IKEv2, SSH, QUIC, WireGuard, Signal, etc.). It is RECOMMENDED
-   to long term completely phase out AKA without forward secrecy.</t>
+   that AKA without forward secrecy be phased out.</t>
```
Comment (seanturner) [inline draft-ietf-emu-aka-pfs-latest.xml:1728]:
```suggestion
   that AKA without forward secrecy be phased out in the long term.</t>

```

Comment (seanturner) [reply draft-ietf-emu-aka-pfs-latest.xml:1728]: @jariarkko would this work?

Comment (emanjon) [reply draft-ietf-emu-aka-pfs-latest.xml:1728]: I am fine with any of three suggestions.

**Comment by emanjon** [comment]
These changes all looks good to me.

Regarding "Added ref for “privacy-friendly identifier". Karl also identified this in his PR and suggested a reference to RFC9190 and and using 3GPP SUCI as an example.

**Review by emanjon — APPROVED** [review]

File: draft-ietf-emu-aka-pfs-latest.xml (hunk 1 above)
Comment (emanjon) [reply draft-ietf-emu-aka-pfs-latest.xml:1728]: Let's change to EAP-AKA methods without without forward secrecy

**Pull request merged**